src/geo/leaflet/leaflet-map-view.js

```diff
@@ -194,6 +194,10 @@ var LeafletMapView = MapView.extend({
     }, this);
   },
 
+  listenOnce: function (name, callback) {
```

Comment:
what is the purpose of this?

Reply:
This method is implemented in gmaps. This is the equivalent leaflet method to pass the tests

Reply:
That's a bit weird (having to add a method just for testing purposes).